iphone/Classes/MediaModule.m

```diff
@@ -2095,8 +2095,14 @@ - (void)imagePickerController:(UIImagePickerController *)picker_ didFinishPickin
     if (cropRect != nil) {
         [dictionary setObject:cropRect forKey:@"cropRect"];
     }
-	
+    
     [self sendPickerSuccess:dictionary];
+    
+#ifndef TI_USE_KROLL_THREAD
+    //TIMOB-24389 : iOS: Appcelerator Titanium App Crashes after some usage time
```

Comment:
Wrote small note about problem. In ticket I think most of the things are explained.